modules/books.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] You are free to remove this as well. If I remove this disabler. I find no error in console.